src/mlpack/methods/decision_tree/decision_tree_regressor_impl.hpp

```diff
@@ -246,15 +245,14 @@ DecisionTreeRegressor<FitnessFunction,
     const size_t maximumDepth,
     DimensionSelectionType dimensionSelector,
     const std::enable_if_t<arma::is_arma_type<
-        typename std::remove_reference<
-        WeightsType>::type>::value>*):
-        splitInfo(std::move(other.splitInfo)),
-        NumericAuxiliarySplitInfo(other),
-        CategoricalAuxiliarySplitInfo(other)   // other info does need to copy
+        std::remove_reference_t<WeightsType>>::value>*)
```

Comment:
```suggestion
        std::remove_reference_t<WeightsType>>::value>*) :
```
Same thing here, other mlpack code has the initialization list ` ` on the preceding line.

Reply:
Done.